android/src/fdroid/play/release-notes/de-DE/default.txt

```diff
@@ -1,9 +1,6 @@
-• neue Kartendaten vom 16. August
-• längere Routen werden nun deutlich schneller berechnet
-• die Suchfunktion ist nun zuverlässiger und schneller
-• reduzierter Speicherplatzverbrauch der Offlinekarten
-• Fehler, dass manche Suchergebnisse nicht auf der Karte angezeigt wurden, wurde behoben
-• verbesserte Darstellung von Militärgebieten und Grenzen
-• barrierefreie Parkplätze werden nun auf der Karte angezeigt
-• verschiedene Icons für Sportarten hinzugefügt
+• Neue Kartendaten vom September
+• Neue Benutzeroberfläche für Kartenschaltflächen
```

Comment:
Better something like "Neue Tasten/Knöpfe/Buttons auf der Kartenoberfläche"

Reply:
Oh I misunderstood the English version. We changed the UI/look of the buttons, am I right (see #3398)? So maybe something like 
"Neue Oberfläche für die Tasten/Knöpfe/Buttons auf der Karte" or 
"Neues Aussehen für die Tasten/Knöpfe/Buttons auf der Karte"